app/helpers/helper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Helpers;
+
+class Helper
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 6</br>Variable "per_page" is not in valid camel caps format